upgrade.php

```diff
@@ -458,6 +458,17 @@ function stringEndsWith($haystack, $needles)
         echo color(' ALREADY EXIST', 'lightgreen') . PHP_EOL;
     }
 
+    // Update users table schema for unique id
+    echo pad(' - Add id field in `users` table');
+    if (false === check_column_exists('users', 'id')) {
+        $sql = "ALTER TABLE `users` ADD COLUMN (
+            `id` INT NOT NULL AUTO_INCREMENT UNIQUE KEY;
```

Comment:
Let's use a BIGINT as column type and move it as first col, so we can later use ID as primary key instead of username